src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUMBER = 5;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int RANDOM_SIDE = 10;
+    private ColorSupplier supplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRundomFigure() {
+        String color = supplier.getRandomColor();

```

Comment:
Instead of using `String` for color, consider using the `Color` enum directly to improve type safety and align with the task requirements.